src/Broadway/EventStore/EventStoreInterface.php

```diff
@@ -28,6 +28,8 @@ public function load($id);
     /**
      * @param mixed                      $id
      * @param DomainEventStreamInterface $eventStream
+     *
+     * @throws EventStoreException
```

Comment:
In my opinion, `DuplicatePlayheadException` should be documented explicitly in this method.

On other hand, some driver-specific exceptions like "Database is down" don't make much sense from `EventStore` client perspective. Someone who uses this interface shouldn't be aware of such implementation details.

Generic `EventStoreException` doesn't tell what happened exactly, so I don't think someone would catch it.

By analogy with Java checked/unchecked exceptions, I'd say `DuplicatePlayheadException` is checked one and "Database is down" is unchecked one. So I'd leave `@throws DuplicatePlayheadException` alone.